#record data
#time
time stamp
#pose
float32 estimateX
float32 estimateY
float32 estimateYaw
float32 obstacleX
float32 obstacleY
float32 obstacleYaw
#vel
float32 estimateVx
float32 estimateVy
float32 obstacleVx
float32 obstacleVy
#dif vel
float32 difVx
float32 difVy
